content/docs/concepts/metric_types.md

```diff
@@ -48,6 +48,10 @@ A _histogram_ samples observations (usually things like request durations or
 response sizes) and counts them in configurable buckets. It also provides a sum
 of all observed values.
 
+Since each observation bucket holds the total count of observations less than or
```

Comment:
How would you phrase it so that it referred to the metrics I would see in a plain text version of the histogram information on a /metrics page that Prometheus is scraping?

What I should see in such case goes beyond being "an implementation detail" and becomes an interface which Prometheus has specific expectations about.  I imagine that implementations may simply maintain a counter for each bucket, and only add them up when they are being scraped.

This page is a conceptual discussion of the _histogram_ and should make it's characteristics clear - certainly after 40-odd years of seeing histograms as graphs this seemed odd, until the 'aha' moment when the point of wishing to blacklist buckets made the reasoning suddenly (blindingly) obvious.